### ISSUE TITLE
Mage spell bug(s)

### ISSUE THREAD

**Issue by LeRutY** [ottconnect/Legion#109]
- [ ] "Ice Lance" (Spell ID: 30455) doesn´t do any damage. http://www.wowhead.com/spell=30455/ice-lance

- [ ] "Blizzard" (ID: 190356) does nothing. http://www.wowhead.com/spell=190356/blizzard

- [ ] "Flurry" (ID: 44614) only hits once, should hit 3 times. http://www.wowhead.com/spell=44614/flurry

If I find more spell bugs for this specific class I´ll report them in this topic.

**Comment by LeRutY** [comment]
NICE, now I get damage using "Fire" spec.

**Comment by ottconnect** [comment]
Can we get a confirmation of what spells I can remove from the list so we can focus on what's not working

**Comment by LeRutY** [comment]
Ice Lance, Blizzard and Flurry still have the same problems, Mobs who got hit by Dragon´s Breath are disoriented for roughly 2 seconds, not for 4.

**Comment by ottconnect** [comment]
Ok but what one is working not the ones that are not working

**Comment by LeRutY** [comment]
None of the spells are working 100% "blizzlike".

**Comment by ottconnect** [comment]
ok but none of the spells work 100% like blizzlike i understand that but whats not working on it, is it the aura, is it the proc handler, is it the same details as before or was their any changes on some of the spell.

The reason i ask is i need to know i'm not using the old repack, so i have 3 core ahead of your core right now, so thats what makes a difference on testing.

**Comment by LeRutY** [comment]
Talking about "Dragon´s Breath" it´s the aura that doesn´t work correctly.
On 1.0.1.9 it should last 4 seconds.

They were no changes on the other 3 spells.

"Ice Lance" has the spell effect, but has no damage value.

"Blizzard" is 0% working, has no effect and does no damage.

"Flurry" has its effect and does damage, but only 1/3 of the damage, because it only fires one missile not 3. 

What do you mean by "Proc Handler"?

**Comment by ottconnect** [comment]
post the string of it coming from the worldserver.exe console

**Comment by ottconnect** [comment]
try this spell for dragon breath this one last 6 sec as per wowhead

http://www.wowhead.com/spell=169843/dragons-breath#see-also-other

in legion many spell have changed numbers you have to bare with me the core is still reading 3.3.5 spell numbers instead of providing the correct updated spell for legion, i'm trying to work hard to find the correct spells i've just tried that and the dragon fire came out in a cone and generated damage

**Comment by ottconnect** [comment]
in legion many spell have changed numbers you have to bare with me the core is still reading 3.3.5 spell numbers instead of providing the correct updated spell for legion, i'm trying to work hard to find the correct spells i've just tried that and the dragon fire came out in a cone and generated damage

**Comment by ottconnect** [comment]
http://www.wowhead.com/spell=228354/flurry

try this one

**Comment by LeRutY** [comment]
http://www.wowhead.com/spell=169843/dragons-breath#see-also-other
It says, that it´s a NPC ability. Also have a cast time, player spell is instant.
http://www.wowhead.com/spell=228354/flurry
It also doesn´t do full damage to mobs. No visual spell effects.

I don´t understand what you want exactly from the worldserver.exe console. Do you want me to do something and then look at the console?

**Comment by ottconnect** [comment]
nothing because i'm thinking your not understanding at all about the new spell system, soi';m just going to do my thing keep posting your errors and we will leave it at that, i've tried both those spells and they seem to work on my side

i need the legion spell numbers from now on not the pre cata spell numbers

**Comment by LeRutY** [comment]
Yes they work but they have wrong and missing effects. Even the default ones are working better.

Why do you think, I post the pre-cata spell numbers and not the Legion ones? For example: "Immolate" http://www.wowhead.com/spell=348/immolate Why is that not the Legion one? On the right side of the page you can see the correct details, you get the spell at level 10 and the tooltip is also like ingame.

I can only post ID´s what idTip says.

**Comment by LeRutY** [comment]
Updated main post, because "Dragon´s Breath" works correctly, the talent "Conflagration" (Spell ID: 205023) stopped the "disorientation" effect.

**Comment by ottconnect** [comment]
So we can throw dragon breath out of the list?
